README.md.template

```diff
@@ -39,6 +39,11 @@ public class BarberView extends FrameLayout {
     @StyledAttr(value = R.styleable.BarberView_stripeColor, kind = Kind.COLOR)
     public int stripeColor;
 
+    // alternately if you use support annotations, the kind can be inferred
+    // @StyledAttr(R.styleable.BarberView_stripeColor)
```

Comment:
I agree but the change-set just checks for the @ColorInt annotation and sets the Kind to Kind.COLOR. I can't immediately think of a way to do this quickly without Kind.COLOR being publicly available too. 
If you want to go down this route, we can probably add some more processing around the Kind.STANDARD, Type.INT case. Meaning, when the kind is STANDARD and the element is an int, instead of setting statement to `getInt(%d, -1)`, we can call a method that determines based on support annotations whether it should be `getInt`, `getColor` or whatever. I'm not in love with this solution though. It add code that does that same thing in multiple places. The current solution keeps the decision making at a higher level and the statement determining code is leaner.